plugins/out_loki/loki.c

```diff
@@ -105,7 +105,7 @@ static flb_sds_t normalize_ra_key_name(struct flb_loki *ctx,
         if (c > 0) {
             flb_sds_cat(name, "_", 1);
         }
-        normalize_cat(rp, name);
+        normalize_cat(rp, &name);
```

Comment:
This change isn't necessary, it actually breaks the function since the pointer passed to the inner frame would not correspond to the `flb_sds_t` but instead to the argument so it needs to be reverted.

Reply:
I misinterpreted this in the diff and didn't realize while looking at the code that this was not a recursive call but a call from an outer function.